### DISCUSSION TITLE
Notice types not unified criteria ("nodeId"s)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#531] [Q&A] [answered]
For example, in the notice types from 1 to 6 (1.json, 2.json, 3.json, 4.json, 5.json, 6.json) in the group "GR-Procedure-Scope-MainClassification" it is not indicated a "nodeId", in the rest of the notice types there is.

Group in 4.json:

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/c6c53189-dcb5-46f7-925a-6457dd670310)

Group in 10.json:

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/254b80dc-3df6-4cca-af4d-c8de7153da59)

This makes it impossible for us to correctly generate the conceptual model. We understand that the notice types should be unified and the same groups should have the same nodeId. Is there an intention to correct this? Or do we have to adapt the SDK to make it work? Since before, the lack of unification of criteria is making it quite difficult for us to use the SDK.

Thanks in advance

**Comment by rouschr** [comment]
Hello,

it is indeed a bit strange that this same group has a nodeId in some notice sub types but not in others.
```
          "id" : "GR-Procedure-Scope-MainClassification",
          "contentType" : "group",
          "nodeId" : "ND-ProcedureMainClassification",
```
As ND-ProcedureMainClassification is not repeatable, you can create it in your conceptual model before going to the physical model.
But we will look into this to see if there is a reason for it to be missing.

**Comment by AlexAS81** [comment]
In addition to not being unified through the nodeId, in one of the notice types it is repeatable (10) and in the other (4) it is not. Does this make sense?

**Reply by rouschr** [reply, marked as answer]
I looked into this and created a ticket, maybe some of these changes will make it into SDK 1.9
